sky/utils/common_utils.py

```diff
@@ -232,11 +232,10 @@ def get_global_job_id(job_timestamp: str,
     return global_job_id
 
 
-def get_unique_task_name(_: 'task_lib.Task') -> str:
-    timestamp = int(time.time())
+def get_unique_task_name(task: 'task_lib.Task') -> str:
+    name = task.name or f'task_{int(time.time())}'
```

Comment:
```suggestion
    name = task.name if task.name is not None else f'task_{int(time.time())}'
```
nit: Always use `if foo is None`.
https://google.github.io/styleguide/pyguide.html#2144-decision

Reply:
I think we should keep it as is. Catching empty strings is also what we want. If the task name is an empty string, we still want to fall back to the timestamp.